vector-config/src/main/res/values/config-settings.xml

```diff
@@ -37,6 +37,7 @@
     <bool name="settings_ignored_users_visible">true</bool>
 
     <!-- Level 1: Labs -->
+    <bool name="settings_labs_deferred_dm_default">true</bool>
```

Comment:
See the doc at the top of this file, you should add another key with `visible` suffix:
```xml
<bool name="settings_labs_deferred_dm_visible">true</bool>
```
and use it in the labs pref file.
Maybe for this very temporary setting you could take this remark as optional.

Reply:
Done https://github.com/vector-im/element-android/pull/7180/commits/dd92bb756a3c331fe6d0af12ed8e66fb00a9b9fc but the "doc" at the beginning of the file does not tell that these fields are "mandatory", it's more a description of the naming convention.

Reply:
OK, thanks 👐